cypress/e2e/awx/administration/wfApprovalsList.cy.ts

```diff
@@ -76,7 +78,7 @@ describe.skip('Workflow Approvals Tests', () => {
     cy.deleteAwxUser(userWFDeny, { failOnStatusCode: false });
   });
 
-  describe('Workflow Approvals - Approve, Deny, Delete', () => {
+  describe.skip('Workflow Approvals - Approve, Deny, Delete', () => {
```

Comment:
```suggestion
  describe.skip('Workflow Approvals - Approve, Deny, Delete', () => {
```
If this test was passing before those changes, we should not skip this one, right?